go/education.html

```diff
@@ -0,0 +1,132 @@
+  //
+  // TODO: Maybe update app index to export the count at a separate URL? Note that
+  //   the CORS hack in Sandstorm core would need updating too.
+  var xhr = new XMLHttpRequest();
+  xhr.open("GET", "https://app-index.sandstorm.io/apps/index.json", false);
+  xhr.send(null);
+
+  if (xhr.status === 200) {
+    appCount = JSON.parse(xhr.responseText).apps.length;
+  }
+})();
+</script>
+
+<section id="web-apps">
+  <h2>Why use Sandstorm?</h2>
+    <h3>University IT teams around the world are installing Sandstorm, an open source dashboard for productivity tools, hosted within the institution.
+
+    Web productivity tools are a few clicks away for educators and students. Enjoy the <script type="text/javascript">document.write(appCount)</script>+ apps in the Sandstorm App Market, all pre-approved, or add custom apps.</h3>
+  
+  <ul>
+    
+    <li class="publish-online">
+      <h3>Publish online</h3>
+      
+    <li class="typeset-papers">
+      <h3>Typeset papers</h3>
+      
+    <li class="organize-projects">
+      <h3>Organize projects</h3>
+      
+    <li class="sync-and-share-files">
+      <h3>Sync and share files</h3>
+      
+    <li class="access-one-place">
+      <h3>Access them all in one place</h3>
+      
+    <li class="use-our-apps">
+      <h3>Use our <script type="text/javascript">document.write(appCount)</script>+ apps, or add custom apps</h3>
+    
+  </ul>
+  <p class="highlight">See our other great apps for calendars, diagrams, forums, and more: <a href="http://apps.sandstorm.io">App Market</a>.</p>
+</section>
+
+<section id="why-it">
+  <h2>Here's why the IT team will love it</h2>
+    <h3>Sandstorm comes with <script type="text/javascript">document.write(appCount)</script>+ great apps for every project. Instructors and students can also package their own apps that anyone can then install.</h3>
+  <ul>
+    
+    <li>
+      <h3>On-prem install for education privacy</h3>
+      <p>Run Sandstorm on hardware that you control. Everything users do on a Sandstorm server stays safely within your institution. Enjoy free dynamic DNS and SSL.</p>
+      
+    <li>
+      <h3>Self-service software</h3>
+      <p>Sandstorm provides simple web access to every user at the institution. When you approve Sandstorm, users do what they want safely within Sandstorm, and they file fewer support tickets.</p>
+
+    <li>
+      <h3>Single-sign-on and web access</h3>
+      <p>Once you enable login via Google, GitHub, or email, every app automatically uses that login provider. LDAP and SAML are to arrive soon.</p>
+    
+    <li>
+      <h3>Custom apps can use the built-in access control</h3>
+      <p>Academic departments need custom apps. As those apps get older, Sandstorm helps you sleep well at night. Sandstorm automatically mitigates against <a href="https://docs.sandstorm.io/en/latest/using/security-non-events/">95% of app security issues, according to our analysis.</a></p>

```

Comment:
"mitigates against" -> "mitigates"

Or you could say "protects against" or "defends against" if you prefer.